### PR TITLE
Remove year from copyright header

### PR DESCRIPTION
**Pull request by frelon**
Signed-off-by: Fredrik Lönnegren <fredrik.lonnegren@suse.com>

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ linters-settings:
     min-complexity: 20 # default is 30 which is too high
   goheader:
     template: |-
-      Copyright © {{ year }} SUSE LLC
+      Copyright © SUSE LLC
```
Comment (Itxaka) [inline .golangci.yml:22]: not sure about thins, doesnt copyright needs to include the year for some weird reason?

Comment (frelon) [reply .golangci.yml:22]: Hehe, we are not alone: https://daniel.haxx.se/blog/2023/01/08/copyright-without-years/

**Review by kkaempf — CHANGES_REQUESTED** [review]
This is still mixing other changes in 😞

**Comment by frelon** [comment]
> This is still mixing other changes in disappointed

Some input reorderings, removed now!

File: .golangci.yml (hunk 1 above)
Comment (fgiudici) [reply .golangci.yml:22]: 🤔  googling around, found some statements that the year (at least the first year of publication) may be needed to compute the duration of the copyright. To make things more complicated, seems different countries may have different rules. Will try to get some advice from the legal dept.

**Comment by frelon** [comment]
Closing after discussions with legal, going with date ranges for files instead.

**Pull request closed without merging**